tox.ini

```diff
@@ -1,19 +1,14 @@
 [tox]
-envlist = py{27,35}-django111-drf{39,latest}, py35-django{20,21,22}-drf{latest}, quality
-
+envlist = py35-django{22,30}, quality
 
 [testenv]
 setenv =
     DJANGO_SETTINGS_MODULE = test_settings
     PYTHONPATH = {toxinidir}
 
 deps =
-    django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
     django22: Django>=2.2,<3.0
-    drf39: djangorestframework<4.0
-    drflatest: djangorestframework
```

Comment:
Why removed the drf tests?

Reply:
Because we are gonna use drflatest anyway, different version was being pinned for python 2.7 and django 1.11, since we have removed those, we don't need testing multiple drf versions.